widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -444,6 +431,40 @@ public void startVisitorVideoView(MediaState visitorMediaState) {
                 .getScreenSharingController(screenSharingCallback);
     }
 
+    private void shouldShowVisitorVideo(CallState state) {
```

Comment:
`should` tells to me that this should return a boolean. Maybe there is another way to name these two methods.